test/conformance/memory/memory_adapter_hip.match

```diff
@@ -1,2 +1,20 @@
-urMemBufferCreateWithNativeHandleTest.Success/AMD_HIP_BACKEND___{{.*}}_
-Segmentation fault
+urMemGetInfoTest.Success/AMD_HIP_BACKEND___{{.*}}
+urMemGetInfoTest.InvalidNullPointerParamValue/AMD_HIP_BACKEND___{{.*}}
+urMemGetInfoTest.InvalidNullPointerParamValue/AMD_HIP_BACKEND___{{.*}}
+urMemGetInfoTest.InvalidNullPointerPropSizeRet/AMD_HIP_BACKEND___{{.*}}
+urMemGetInfoTest.InvalidNullPointerPropSizeRet/AMD_HIP_BACKEND___{{.*}}
+urMemImageCreateTest.InvalidSize/AMD_HIP_BACKEND___{{.*}}
```

Comment:
I'm not sure how to deal with this thing. Perhaps we need to query the max image width depth etc when a device is initialized so we can check vals against this at Image creation